stacks/apps/feeder.yml

```diff
@@ -262,17 +262,31 @@ Resources:
                 Resource:
                   - !GetAtt DefaultJobQueue.Arn
                   - !GetAtt FixerCallbackQueue.Arn
-                Sid: AllowSendAndReceive
+                Sid: AllowShoryuken
             Version: "2012-10-17"
-          PolicyName: Queues
+          PolicyName: AppQueues
+        - PolicyDocument:
```

Comment:
Adds missing SQS permissions for Feeder-specific Announce queues, that Feeder reads from.